corehq/motech/forms.py

```diff
@@ -132,12 +132,15 @@ def helper(self):
             crispy.Field('plaintext_password'),
             crispy.Field('client_id'),
             crispy.Field('plaintext_client_secret'),
-            crispy.Fieldset(
-                _("Api Auth Settings"),
-                crispy.Field('auth_preset'),
-                crispy.Field('token_url'),
-                crispy.Field('refresh_url'),
-                twbscrispy.PrependedText('pass_credentials_in_header', ''),
+            crispy.Div(
+                crispy.Fieldset(
+                    _("OAuth 2.0 Settings"),
+                    crispy.Field('auth_preset'),
+                    crispy.Field('token_url'),
+                    crispy.Field('refresh_url'),
+                    twbscrispy.PrependedText('pass_credentials_in_header', ''),
+                ),
+                id="div_id_oauth_settings",
```

Comment:
Is this ID to show/hide the settings if we're using custom URLs or a preset? Do you need to update `$authPreset.change` to use this ID instead of the IDs of each of the fields?

Reply:
No, that ID is to show the whole section if you select an OAuth 2.0 grant type (client or password
![Peek 2021-11-26 09-02](https://user-images.githubusercontent.com/146896/143592355-829d97dd-93f8-4f34-89b6-2807b11b706b.gif)
)

Reply:
Ah! Nice!